.buildkite/pipeline.yml

```diff
@@ -401,32 +435,36 @@ steps:
         account_ids: "436617320021"
         no-include-email: true
         region: "us-west-2"
-      docker#v2.1.0:
+      docker#v3.2.0:
+        shell: ["/bin/bash", "-i", "-e", "-c"]
         debug: true
-        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:ubuntu18_2-3"
+        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:ubuntu18_2-4"
         propagate-environment: true
         workdir: /data/job
     env:
       OS: "ubuntu-18.04"
       PKGTYPE: "deb"
     timeout: 60
 
-  - command: | # macOS Mojave Package Builder
-        echo "--- :arrow_down: Downloading build directory"
-        buildkite-agent artifact download "build.tar.gz" . --step ":darwin: Mojave Build"
-        tar -zxf build.tar.gz
-        echo "+++ :microscope: Starting package build"
-        ln -s "$(pwd)" /data/job
-        cd /data/job/build/packages
-        bash generate_package.sh brew
-    label: ":darwin: Mojave Package Builder"
+  - label: ":darwin: Mojave Package Builder"
+    command:
+      - "tar -xzf /Network/NAS/MAC_FLEET/BUILDKITE/artifacts/${ANKA_MOJAVE_TEMPLATE}-${BUILDKITE_PIPELINE_SLUG}-${BUILDKITE_BUILD_ID}.tar.gz"
```

Comment:
Why use the NAS vs buildkite artifact?

Reply:
Much faster and eliminates data usage from the local network out (Nick V requested this)